content/2016.12.how-to-develop-a-webapp-with-Kinto.md

```diff
@@ -0,0 +1,235 @@
+Title: How to develop a WebApp with Kinto
+Date: 2016-12-01 10:20
+Slug: develop-a-chat-webapp-with-kinto
+Authors: Enguerran Colson
+Summary: An example of how Kinto handles a webapp's data storage.
+
+## Our goal
+
+Hi folks, today, we are gonna create a simple WebApp that allows anyone to chat in a room accessible through a URL.
+
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_4340C716162DF03B2B97575BFA438C542185694D96D86DB6EB44B989957A3ADE_1480537293391_file.png)
```

Comment:
Can I suggest we version post images in the repository? That would help reviewing too :)